ui/cypress/integration/common-features/slow_query/list_spec.js

```diff
@@ -0,0 +1,211 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
```

Comment:
How about organizing the code by functional modules. So that we can reduce some directory nesting and `/__image_snapshots__` that generated by https://github.com/meinaart/cypress-plugin-snapshots can also be organized by functional modules.

Like:
```
/slow_query/list.spec.js
/slow_query/list.compat_spec.js
...
```

Reply:
Sure, I will update the organization.